src/megaapi_impl.cpp

```diff
@@ -879,19 +879,29 @@ char *MegaNodePrivate::getPublicLink(bool includeKey)
         return NULL;
     }
 
-    string strlink = "https://mega.nz/#";
-    strlink += (type ? "F" : "");
+    string strlink = "https://mega.nz/";
```

Comment:
It might worth to move this part into an static method in `std::string& MegaClient::getPublicLink(type, ph, key)` and use it as well in `megacli` at `exportnode_result()`

Reply:
Also, check `megacli` can log into folder links of new format. Download file links should work fine once the `openfilelink()` is adjusted.